### PR TITLE
[MIG] mail_tracking: Migration to 14.0

### PR DESCRIPTION
**Pull request by angelJuaGar**

### CODE REVIEW

**Comment by SimoneVagile** [comment]
@angelJuaGar I can't create a PR to your branch, can you cherry-pick this commit?

[https://github.com/SimoneVagile/social/commit/5a758db57e09cfaf1dd3d8c15155a09923f68e22](https://github.com/SimoneVagile/social/commit/5a758db57e09cfaf1dd3d8c15155a09923f68e22)

**Comment by angelJuaGar** [comment]
@AreebSiddiqiIR @SimoneVagile @pedrobaeza @moylop260
The migration is ready

**Comment by moylop260** [comment]
@angelJuaGar 

Could you run `pre-commit run --all`, please?
In order to auto-fix the red status of the ci

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+mock
```
Comment (moylop260) [inline requirements.txt]: Please, use `test-requirements.txt` file for libraries used only in unittest since that it is not used in the models

**Review by moylop260 — APPROVED** [review]
👍

**Review by SimoneVagile — APPROVED** [review]

**Pull request merged**